src/rust/src/x509/verify.rs

```diff
@@ -54,6 +55,20 @@ pyo3::create_exception!(
     pyo3::exceptions::PyException
 );
 
+macro_rules! policy_builder_set_once_check {
```

Comment:
This is a nice cleanup -- you can split it into its own PR and we can merge easily.

Reply:
I have created the PR.